transmart-core-db/grails-app/services/org/transmartproject/db/log/AccessLogService.groovy

```diff
@@ -3,14 +3,25 @@
 package org.transmartproject.db.log
 
 import grails.transaction.Transactional
+import groovy.json.JsonBuilder
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import org.transmartproject.core.log.AccessLogEntryResource
 import org.transmartproject.core.users.User
 
 @Transactional
 class AccessLogService implements AccessLogEntryResource {
 
+    static final Logger log = LoggerFactory.getLogger(AccessLogService.class)
+
     @Override
     AccessLogEntry report(Map<String, Object> additionalParams = [:], User user, String event) {
+        log.trace(new JsonBuilder([
```

Comment:
In other places, we try to stick to Jackson for (de)serialising JSON, e.g., 
```java
new ObjectMapper.writeValueAsString(value)
```
(or GSON for streaming serialisation... and trying to get of the Grails specific JSON converters...)